### DISCUSSION TITLE
一些好的资源 - 计算机学习指南

### DISCUSSION THREAD

**Discussion by giscus** [meowrain/doc-for-sxau#43] [Ideas]
# 一些好的资源 - 计算机学习指南

计算机学习指南，这里有从网络上收集的计算机各种方面的知识技能，可供查阅和学习

https://cs.meowrain.cn/%E6%96%87%E6%A1%A3%E6%94%B6%E9%9B%86/Collections-Document-CsStudent/

**Comment by meowrain** [comment]
https://oi-wiki.org/math/
算法竞赛，OI
